checkstyle/build.gradle.kts

```diff
@@ -28,6 +28,8 @@ gradlePlugin {
       implementationClass = "org.projectnessie.buildtools.checkstyle.CheckstyleHelperPlugin"
     }
   }
+  vcsUrl.set("https://github.com/projectnessie/nessie/")
```

Comment:
Should it be `https://github.com/projectnessie/gradle-build-plugins`?

Reply:
It doesn't really matter - we don't publish these plugins to `plugins.gradle.org`.

Reply:
then why both with URLs? :thinking: